app/src/main/java/uk/ac/ebi/atlas/experimentpage/tsneplot/TSnePlotDao.java

```diff
@@ -50,7 +50,11 @@ public TSnePlotDao(NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
 
     private static final String SELECT_T_SNE_PLOT_WITH_CLUSTERS_STATEMENT = "SELECT c.cell_id,c.x,c.y, g.value as " +
             "cluster_id " + "FROM scxa_cell_group g " + "JOIN scxa_cell_group_membership m ON " + "g.id = m" +
-            ".cell_group_id AND " + "g.experiment_accession = :experiment_accession AND " + "g.variable = :variable " + "RIGHT JOIN scxa_coords c ON " + "m.cell_id=c.cell_id AND " + "m.experiment_accession=c.experiment_accession " + "WHERE c.method=:method AND c.parameterisation->0->>:parameter_name=:parameter " + "AND c.experiment_accession=:experiment_accession";
+            ".cell_group_id AND " + "g.experiment_accession = :experiment_accession AND " + "g.variable = :variable " +
+            "RIGHT JOIN scxa_coords c ON " + "m.cell_id=c.cell_id AND " +
```

Comment:
I could see more space after `AND " + "g.variable = :variable " +`. Please double check.

Reply:
Sorry I didnot see any more space in the query. Would you like to confirm?